src/sidebar/components/button.js

```diff
@@ -105,17 +120,19 @@ Button.propTypes = {
   /** Is this button currently in an "active"/"on" state? */
   isActive: propTypes.bool,
 
-  /** callback for button clicks */
-  onClick: propTypes.func,
-
-  /** optional inline styling  */
-  style: propTypes.object,
-
   /**
    * `title`, used for button `title`, is required unless `buttonText` is present
    */
   title: requiredStringIfButtonTextMissing,
 
+  /**
+   * One of: "toggle", "group".
+   *  "toggle" type uses the `aria-pressed` attribute in conjunction with `isActive`.
+   *  "group" type uses the `aria-expanded` attribute in conjunction with `isActive`.
```

Comment:
This comment explains how the component interprets these two possible values, but not what they accomplish. In what contexts would one use `toggle` or `group`? I mean, I have a general sense (especially with `toggle`), but not a complete one.